src/views/GovernanceTaskList/index.tsx

```diff
@@ -42,11 +43,81 @@ const GovernanceTaskList = () => {
   );
 
   const intakeStatus = intakeStatusFromIntake(systemIntake);
-  const intakeLink = chooseIntakeLink(systemIntake, intakeStatus);
+  const intakeLink = () => {
+    const link = chooseIntakeLink(systemIntake, intakeStatus);
```

Comment:
I think this is likely closer to a URL than a link? Or maybe a path? Naming nit... not necessary to change.